docs/pages/sensitive-data.rst

```diff
@@ -0,0 +1,37 @@
+Logging sensitive data
+======================
+
+Jupyter Telemetry offers flexibility when logging and handling sensitive data. 
+
+Since events may include data with varying degrees of sensitivity, Jupyter Telemetry uses a multi-level security approach. It exposed four levels of sensitivity:
```

Comment:
Is there a default level when unspecified, or is level always required to be explicit for every property?

Reply:
Right now, the default is `unclassified` if level is not given, but I think it's an open question before merging this. I actually think it makes sense to require the level to be explicit in each property... what are your thoughts?

Reply:
I think it would be nice to have it be required.